File: web/hasMaxLength.js
function hasMaxLength($value, $maxLength) {
    if ($value.isLiteral()) {
        return $value.getLex().length <= $maxLength.getLex();
    } else if ($value.isURI()) {
        return $value.getUri().length <= $maxLength.getLex();
    } else { // Blank node
        return false;
    }
}

File: web/has_max_length.py
def has_max_length(_value, _maxLength):
    if _value.isLiteral():
        return len(_value.getLex()) <= int(_maxLength.getLex())
    elif _value.isURI():
        return len(_value.getUri()) <= int(_maxLength.getLex())
    else:
        return False